src/main/java/com/browserstack/demos/BrowserStack.java

```diff
@@ -2,17 +2,31 @@
 
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.util.HashMap;
 
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.MutableCapabilities;
 import org.openqa.selenium.remote.RemoteWebDriver;
 
 public class BrowserStack {
 
     public static void main(String args[]) throws MalformedURLException {
-        ChromeOptions options = new ChromeOptions();
+        HashMap<String, Object> bstackOptions = new HashMap<String, Object>();
+        bstackOptions.put("os", "Windows");
+        bstackOptions.put("osVersion", "7");
+        bstackOptions.put("sessionName", "BStack automate-java");
+        bstackOptions.put("buildName", "browserstack-build-1");
+        bstackOptions.put("source", "automate-java:sample-selenium-4:v1.0");
+        bstackOptions.put("projectName", "My Awesome App");
+        bstackOptions.put("debug", true);
+        bstackOptions.put("seleniumVersion", "3.12.0");
+
+        MutableCapabilities capabilities = new MutableCapabilities();
+        capabilities.setCapability("browserstack.use_w3c", true);
```

Comment:
remove this

Reply:
capabilities.setCapability("browserstack.use_w3c", true);
This one??

Reply:
yes